content/index.html.haml

```diff
@@ -24,6 +24,9 @@ homepage: true
     ['redhat.png',
     'RedHat, Inc.',
     'https://redhat.com'],
+    ['github.png',
+    'GitHub, Inc.',
+    'https://redhat.com'],
```

Comment:
Why it's redhat.com?

Reply:
Uhh yeah mistake. I’ll close the pull request. Fix and PR again

Reply:
Uhh yeah mistake. I’ll close the pull request. Fix and PR again

Reply:
Uhh yeah mistake. I’ll close the pull request. Fix and PR again

Reply:
You don't need to close the PR. You can ammend the commit.